audio-worklet/src/bitcrusher-demo.js

```diff
@@ -123,13 +189,16 @@ class BitcrusherDemo {
     // Play song, running samples through a bitcrusher under user control.
     this.song_ =
```

Comment:
It's hard to refer `song_` below. Please rename it something implies `bufferSource`.